examples/references.bib

```diff
@@ -48,6 +48,24 @@ @misc{carpenter2016hierarchical
   year          = {2016},
   publisher     = {Technical report. Retrieved from https://mc-stan. org/users/docum entat ion~\ldots{}}
 }
+@article{collinswilson2019,
+  title         = {Ten simple rules for the computational modeling of behavioral data},
+  author        = {Wilson, Robert C and Collins, Anne GE},
+  editor        = {Behrens, Timothy E},
+  volume        = 8,
+  year          = 2019,
+  month         = {nov},
+  pub_date      = {2019-11-26},
+  pages         = {e49547},
+  citation      = {eLife 2019;8:e49547},
+  doi           = {10.7554/eLife.49547},
+  url           = {https://doi.org/10.7554/eLife.49547},
```

Comment:
I think only one of them should be present, I'd keep the doi only. Both are rendered but provide the same info so it looks duplicated: https://pymcio--410.org.readthedocs.build/projects/examples/en/410/case_studies/reinforcement_learning.html#references

Reply:
Removed in https://github.com/pymc-devs/pymc-examples/pull/410/commits/053036a309e498461f25e0f6748c281334c1f706